src/components/navbar/index.tsx

```diff
@@ -66,20 +75,34 @@ const Navbar: React.FC = (): JSX.Element => {
     setComponentViewSave("TutorialSelector");
   };
 
+  const openTutorialEditor = (): void => {
+    setComponentView("TutorialEditor");
+    setComponentViewSave("TutorialEditor");
+  };
+
   return (
     <NavWrapper>
-      <NavLogo to="/" onClick={openTutorialComponent}>
+      <NavLogo
+        to="/"
+        onClick={openTutorialComponent}
+        isSelected={componentView === "TutorialComponent"}
```

Comment:
To avoid console error you could try setting it to a string.
Ex: {(componentView === "TutorialComponent").toString()}